r/tests/testthat/helper-arrow.R

```diff
@@ -56,6 +56,18 @@ test_that <- function(what, code) {
   })
 }
 
+# backport of 4.0.0 implementation
+suppressWarnings <- function (expr, classes = "warning") {
+  withCallingHandlers(
+    expr,
+    warning = function(w) {
+      if (inherits(w, classes)) {
+        invokeRestart("muffleWarning")
+      }
+    }
+  )
+}
```

Comment:
Could we wrap this in an `if(getRversion() < "4.0.0") {...}` So that we can continue using the official `suppressWarnings()` (with the additions of tryInvokeRestart, etc.) for versions that support it?